packages/fxa-settings/src/pages/Pair/AuthTotp/index.stories.tsx

```diff
@@ -0,0 +1,33 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+import React from 'react';
+import AuthTotp from '.';
+import AppLayout from '../../../components/AppLayout';
+import { Meta } from '@storybook/react';
+import { LocationProvider } from '@reach/router';
+import { MOCK_ACCOUNT } from '../../../models/mocks';
+import { MozServices } from '../../../lib/types';
+
+export default {
+  title: 'pages/Pair/AuthTotp',
+  component: AuthTotp,
+} as Meta;
+
+const storyWithProps = ({ ...props }) => {
+  const story = () => (
+    <LocationProvider>
```

Comment:
Do we need `LocationProvider` here?

Reply:
actually I think we might not!  Thank you!